_docs/BASE.md.jinja2

```diff
@@ -0,0 +1,6 @@
+{% set modules = {'modules/lx-autoscale': 'Linux Autoscaling Group', 'modules/lx-instance': 'Linux Instance', 'modules/win-autoscale':'Windows Autoscaling Group', 'modules/win-instance': 'Windows Instance'} %}
```

Comment:
Let's name this file `MAIN.md.jinja` since it is output to `MAIN.md` when it rendered...